src/content/protocols/ethena/usde.md

```diff
@@ -0,0 +1,329 @@
+
+> Autonomy score: High
+
+## Exit Window
+
+The actions of changing the minter role, blacklisting `USDe` stakers and seizing their funds are not protected by an _Exit Window_ and can be executed immediately, thus could harm the user immediately.
+
+> Exit Window score: High
+
+## Accessibility
+
+The user interactions with the protocol can be divided into two parts: the minting and redeeming of `USDe` and the staking of `USDe`.
+
+Users can host a frontend to mint and redeem `USDe` from https://github.com/ethena-labs/ethena-minting-client. For this action the _Accessibility_ risk is _Medium_.
+
+For staking `USDe` the protocol does not provide a frontend to self host. The staking is only possible through the Ethena frontend. For this action the _Accessibility_ risk is _High_.
+
+> Accessibility score: High
+
+## Conclusion
+
+The USDe stablecoin protocol achieves High centralization risk scores for its _Upgradeability_, _Autonomy_, _Exit Window_ and _Accessibility_ dimensions. This would mean that the protocol ranks Stage 0, however the funds are stored in centralzed custody and thus is classified as _Others_.
+
+# Reviewer's Notes
+
+No additional notes.
+
+# Protocol Analysis
+
+![Architecture](../diagrams/Ethena.png)
+
+## USDe Token
+
+The owner of the `USDe` contract is the [Dev Multisig](#security-council). The [Dev Multisig](#security-council) can set any minter address, which can increase the `USDe` supply and mint `USDe` tokens to any address. The multisig is trusted to not abuse this power. The minter address is set to the `EthenaMinting` contract, which requires a transfer of accepted collateral, before minting `USDe`.
+
+## Minting and Redeeming USDe
+
+The Ethena stablecoin protocol allows whitelisted users to mint and redeem `USDe` through a flow that combines off-chain quoting and on-chain order submission with final execution by Ethena servers using EOAs with roles `MINTER_ROLE` and `REDEEMER_ROLE` (see [table](#role-permission-inside-ethenaminting)).
+
+The process begins off-chain, where a whitelisted user interacts with an Ethena-hosted pricing server to request a quote. This quote reflects the amount of collateral to be deposited and the expected `USDe` to be received. Using this information, the user constructs and signs an EIP-712 order. The signed order includes the benefactor address, collateral asset and amount of `USDe` minted.
+
+A central server operated by the Ethena team validates this order. It ensures the collateral is eligible, that the order does not destabilize the system, and that the minting or redemption action complies with both global and per-asset block-level caps. These caps are enforced to prevent drastic changes in `USDe` supply and drastic collateral exposures, which could also destabilize the system.
```

Comment:
I would emphasize that those are not strict "enforcements", since this is not onchain